File: CURSO PHP POO/CURSO POO PHP/Aula12/Peixe.php
<?php
require_once 'Animal.php';
class Peixe extends Animal{
    private $corEscama;
    
    public function locomover() {
        echo "<p>Nadando</p>";
    }
    public function alimentar() {
        echo "<p>Comendo substâncias</p>";
    }
    public function emitirSom() {
        echo "<p>Peixe não faz som</p>";
    }
    public function soltarBolha() {
        echo "<p>Soutou uma bolha °°°°°000</p>";
    }
    function getCorEscama() {
        return $this->corEscama;
    }

    function setCorEscama($corEscama) {
        $this->corEscama = $corEscama;
    }


}

File: EXERCÍCIOS JAVA E JAVA POO/Aula12_POO/src/aula12_poo/Peixe.java
package aula12_poo;
public class Peixe extends Animal{
   private String corEscama; 

    @Override
    public void locomover() {
        System.out.println("Nadando");
    }

    @Override
    public void alimentar() {
        System.out.println("Comendo Substâncias");
    }

    @Override
    public void emitirSom() {
        System.out.println("Peixe não faz Som");
    }
    
    public void soltarBolha(){
        System.out.println("Soutou uma Bolha°°°°°0000");
    }

    public String getCorEscama() {
        return corEscama;
    }

    public void setCorEscama(String corEscama) {
        this.corEscama = corEscama;
    }
    
}
